lib/manageiq/automation_engine/logger.rb

```diff
@@ -1,6 +1,20 @@
 module ManageIQ
   module AutomationEngine
     class Logger < ManageIQ::Loggers::Base
+      attr_reader :wrapped_logger
+
+      def initialize(*args, **kwargs)
+        @wrapped_logger = kwargs.fetch(:wrapped_logger)
+        super(*args, **kwargs.except(:wrapped_logger))
+      end
```

Comment:
Alternatively we can enforce the wrapped_logger parameter with:

```suggestion
      def initialize(*args, wrapped_logger:, **kwargs)
        @wrapped_logger = @wrapped_logger
        super(*args, **kwargs)
      end
```